tests/logictest/suites/base/06_show/06_0009_show_shares

```diff
@@ -1,64 +1,63 @@
-statement ok
-DROP DATABASE IF EXISTS show_shares;
+-- statement ok
+-- DROP DATABASE IF EXISTS show_shares;
 
-statement ok
-DROP SHARE IF EXISTS test_share;
+-- statement ok
+-- DROP SHARE IF EXISTS test_share;
 
-statement ok
-CREATE DATABASE show_shares;
+-- statement ok
+-- CREATE DATABASE show_shares;
 
-statement ok
-USE show_shares;
+-- statement ok
+-- USE show_shares;
 
-statement ok
-DROP TABLE IF EXISTS test_tb;
+-- statement ok
+-- DROP TABLE IF EXISTS test_tb;
 
-statement ok
-create table test_tb (a int);
+-- statement ok
+-- create table test_tb (a int);
 
-statement ok
-create share test_share comment = 'comment';
+-- statement ok
+-- create share test_share comment = 'comment';
 
-statement ok
-alter share test_share add tenants = x;
+-- statement ok
+-- alter share test_share add tenants = x;
 
-statement ok
-grant USAGE on DATABASE show_shares TO SHARE test_share;
+-- statement ok
+-- grant USAGE on DATABASE show_shares TO SHARE test_share;
 
-statement ok
-grant SELECT on TABLE show_shares.test_tb TO SHARE test_share;
+-- statement ok
+-- grant SELECT on TABLE show_shares.test_tb TO SHARE test_share;
 
-statement query RTTTTTT
-show shares;
+-- statement query RTTTTTT
+-- show shares;
 
-----
-$DATE_IN_SHARE OUTBOUND test_share show_shares test_tenant x comment
+-- ----
+-- $DATE_IN_SHARE OUTBOUND test_share show_shares test_tenant x comment
```

Comment:
I think `$DATE_IN_SHARE` is meaningless, ideally, it should be

```sql
mysql> show shares;
+--------------------------------+----------+------------+---------------+-------+------+---------+
| Created_on                     | Kind     | Name       | Database_name | From  | To   | Comment |
+--------------------------------+----------+------------+---------------+-------+------+---------+
| 2022-08-24 01:46:04.191485 UTC | OUTBOUND | test_share | show_shares   | admin | x    | comment |
+--------------------------------+----------+------------+---------------+-------+------+---------+
```

Reply:
It's regexp match because the result may be different.

Reply:
got it

Reply:
> I think `$DATE_IN_SHARE` is meaningless, ideally, it should be

There are two kinds of Date regex used in stateless test, It is really meaningless. You can change in dir logictest/regex_type.py, all ref test suites need changing at the same time.